pandas/io/formats/style_render.py

```diff
@@ -497,119 +497,131 @@ def _translate_body(
 
         body = []
         for r, row_tup in enumerate(self.data.itertuples()):
-            if r >= max_rows:  # used only to add a '...' trimmed row:
-                index_headers = [
-                    _element(
-                        "th",
-                        f"{row_heading_class} level{c} {trimmed_row_class}",
-                        "...",
-                        not self.hide_index_[c],
-                        attributes="",
-                    )
-                    for c in range(self.data.index.nlevels)
-                ]
-
-                data = [
-                    _element(
-                        "td",
-                        f"{data_class} col{c} {trimmed_row_class}",
-                        "...",
-                        (c not in self.hidden_columns),
-                        attributes="",
-                    )
-                    for c in range(max_cols)
-                ]
+            if r not in self.hidden_rows:
```

Comment:
only added this line to this method and all code below it was indended one tab.

Reply:
might be worth splitting up translate_body to some function calls to make it simpler to grok (future)

Reply:
i will finish making this clean up after #43686